### ISSUE TITLE
BUG OEmbed shortcodes not being parsed by tinymce plugin

### ISSUE THREAD

**Issue by tractorcow** [silverstripe/silverstripe-asset-admin#692]
When the following shortcode is embedded into a page, it initially appears correctly, but on reload this will be replaced with a literal string that isn't turned into an embedded image.

```
[embed url="https://www.w3schools.com/css/img_fjords.jpg" thumbnail="https://www.w3schools.com/css/img_fjords.jpg" class="leftAlone ss-htmleditorfield-file embed" width="100" height="100"]https://www.w3schools.com/css/img_fjords.jpg[/embed]
```

Tested on the ss4 demo site.

# PRs

 - [ ] https://github.com/silverstripe/silverstripe-asset-admin/pull/695

**Comment by flamerohr** [comment]
@tractorcow I'm having trouble reproducing this issue (includes using the same image url, and pasting that same shortcode in the "code" button->modal)

**Comment by tractorcow** [comment]
To reproduce:

 - Go to https://ss4-demo.silverstripe.org/admin/pages/edit/show/6672 (login admin / password)
 - Insert media
 - Add url `https://www.w3schools.com/css/img_fjords.jpg`
 - Notice media is inserted safely
 - Save and publish
 - Reload page
 - Notice page has shortcode displayed literally, and not parsed anymore

**Comment by tractorcow** [comment]
Note: Going to "view source", pressing "ok" (without making any changes) fixes it temporarily.

**Comment by tractorcow** [comment]
To reproduce this locally:

 - Create a page
 - Go to "view source"
 - Copy in content from https://gist.github.com/tractorcow/1c225f7c88fbed9e2be7897380b9b27c
 - save and publish
 - Reload

**Comment by tractorcow** [comment]
I found this in the code
```
// @todo - replace with `lib/ShortcodeSerialiser` library
```

Probably will do exactly this to fix :)

**Comment by tractorcow** [comment]
@flamerohr fixed in #695
